.changeset/moody-roses-explain.md

```diff
@@ -0,0 +1,15 @@
+---
+"@rocket/building-rollup": patch
+"@rocket/cli": patch
+"@rocket/drawer": patch
+"@rocket/eleventy-rocket-nav": patch
+"@mdjs/core": patch
+"@rocket/search": patch
```

Comment:
can we have all the versions bump only packages in a separate changeset?

otherwise, they all will get the search improvements text added into their changeset

Reply:
this is because of the dependencies